docs/baseplate/message_queue.rst

```diff
@@ -10,11 +10,11 @@ This module provides a thin wrapper around POSIX Message queues.
 
    There are also various limits on the sizes of queues:
 
-   * The ``msgqueue`` rlimit limits the amount of space the user can use on
+   * The ``msgqueue`` ``rlimit`` limits the amount of space the user can use on
      message queues.
-   * The ``fs.mqueue.msg_max`` and ``fs.mqueue.msgsize_max`` sysctls limit the
-     maximum number of messages and the maximum size of each message respectively
-     which a queue can be configured to have.
+   * The ``fs.mqueue.msg_max`` and ``fs.mqueue.msgsize_max`` ``sysctls`` limit
```

Comment:
nit: ``sysctls`` may not be worth monospacing, since the command name is ``sysctl``. It's _possible_ that this could make a reader pause for a second to ponder intent.